tests/test_cache_manager.py

```diff
@@ -445,7 +445,7 @@ def test_nested_directories(cm_client, tmpdir):
                                 "num_unhandled_errors": "0",
                                 "event": ["1", "256", "0", "'xul__4byte.symc'"],
                                 "event_flags": ["<flags.CREATE: 256>"],
-                                "flags_list": "'flags.CREATE'",
+                                "flags_list": "'256'",
```

Comment:
enum serialization behavior changed in 3.11 https://docs.python.org/3/whatsnew/3.11.html#enum